openmc/settings.py

```diff
@@ -163,7 +163,7 @@ class Settings:
     statepoint : dict
         Options for writing state points. Acceptable keys are:
 
-        :batches: list of batches at which to write source
+        :batches: list of batches at which to write state
```

Comment:
Thanks for spotting this, I agree ```source``` is not the right word here and ```state``` is certainly better.

What do people think about going for ```statepoint file``` as an alternative?

```suggestion
        :batches: list of batches at which to write statepoint files
```

Reply:
I think `statepoint files` is more clear.